package.json

```diff
@@ -1,14 +1,17 @@
 {
   "scripts": {
+    "_check-links": "make check-internal-links",
     "build-and-serve": "npm run build && npm run serve",
     "build:preview": "hugo --cleanDestinationDir -DFE --minify --baseURL \"${DEPLOY_PRIME_URL:-/}\"",
     "build:production": "hugo --cleanDestinationDir --minify",
     "build": "hugo --cleanDestinationDir -e dev -DFE",
-    "check-links": "make check-internal-links",
-    "get-submodules": "git submodule update --init --recursive --depth 1",
-    "postbuild:preview": "npm run check-links",
-    "preinstall": "npm run get-submodules",
-    "serve": "netlify dev -c \"hugo serve --minify -DFE -w\""
+    "check-links": "npm run _check-links",
+    "postbuild:preview": "npm run _check-links",
+    "precheck-links": "npm run build",
```

Comment:
Interesting i didn't know you could `pre` any  script. I'd assumed that things like `preinstall` were built-ins.

Reply:
Yes, it's very useful 😄